src/v3/src/components/LaunchAuthenticatorButton/LaunchAuthenticatorButton.tsx

```diff
@@ -65,6 +65,11 @@ const LaunchAuthenticatorButton: UISchemaElementComponent<{
     }
     onSubmitHandler({
       step,
+      isActionStep: true,
```

Comment:
@glenfannin-okta would we always want this to be an `actionStep`?  Or only when `rememberMe` is `true`?

Reply:
I see you changed it, I think the updated way you have it is good, just in case there is some weird reason why it should not be treated as an action step when the rememberMe option is `false`.